src/field.rs

```diff
@@ -798,6 +798,8 @@ impl FieldElement {
     /// Given a nonzero field element, compute its inverse.
     /// The inverse is computed as self^(p-2), since
     /// x^(p-2)x = x^(p-1) = 1 (mod p).
+    ///
+    /// XXX should we add a debug_assert that self is nonzero?
```

Comment:
Only concern is that it may be useful to have 0.inverse() return zero, for similar reasons as it's useful to have the invsqrt function return 0 on 0 in Decaf...